src/Redux/Reducers-Redux/authSlice.js

```diff
@@ -0,0 +1,93 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+import {notifySuccess,notifyError} from "Utilities/Notifications";
+
+const initialState={
+    currentUser:{},
+    loading:false
+    }
+
+export const login=createAsyncThunk("auth/login",async(userDetails)=>{
+try {
+    const response=await axios.post(`/api/auth/login`,{
+    username:userDetails.username,
+    password:userDetails.password,
+    });
+    return response.data;
+} catch (error) {
+  console.log(error.response.data.errors[0]);  
```

Comment:
Instead of console logging the error, also have a toast/alert so that user can know what went wrong.

Reply:
Nice suggestion will implement it thanks